### ISSUE TITLE
Fix potential vulnerability CVE-2023-5217 on FFMPEG

### ISSUE THREAD

**Issue by Suaro** [bytedeco/javacpp-presets#1431]
According to https://nvd.nist.gov/vuln/detail/CVE-2023-5217 someone found vulnerability issue in libvpx version prior to 1.13.0 which allows remote attacker to crash or inject arbitrary code into an application. This issue was found in Chrome first, but according to other reports like https://access.redhat.com/security/cve/cve-2023-5217 or https://www.rezilion.com/blog/the-cve-2023-5217-deja-vu-another-actively-exploited-chrome-vulnerability-affecting-a-webm-project-library-libvpx/ ffmpeg could be potentially affected.

Google published fix version ([1.13.1](https://github.com/webmproject/libvpx/releases/tag/v1.13.1))  ABI compatible with version 1.13.0. I don't really know if it could affect to FFMPEG preset, but due to criticality of the issue I think it would be good if you upgrade FFMPEG preset at any rate. Currently, javacpp-preset is using 1.13.0 version, so I think fix should be easy.

Regards!

**Comment by saudet** [comment]
Please open a pull request!

**Comment by Suaro** [comment]
Hi @saudet, I've just created pull request for ffmpeg https://github.com/bytedeco/javacpp-presets/pull/1436. I was able to compile it locally and it seems no breaking changes.

I checked Leptonica version of libwebp (prior to 1.3.2 is vulnerable) and it seems it's in latest version, so, apparently it's not affected. In case of OpenCV, I found they updated libwebp to 1.3.2 2 months ago, so, probably latest version of OpenCV preset is already updated. Anyway, in following releases, it will be fixed.

Regards!
